conf/messages

```diff
@@ -602,7 +633,6 @@ email-whitelisting.problem.bullet4=the trial has ended
 email-whitelisting.problem.p3=You’ll need to email HMRC at <strong class="bold">ddcw-info-cgtpd-g@digital.hmrc.gov.uk</strong> to request access to the service.
 email-whitelisting.problem.p4=You should include your name.
```

Comment:
We don't need these messages anymore now I don't think - we don't do anything with whitelisting anymore. Not sure if there are templates lurking around somewhere which are still using these messages

Reply:
there is no usage so I deleted them